types/debug/index.d.ts

```diff
@@ -30,6 +30,13 @@ declare namespace debug {
         skips: RegExp[];
 
         formatters: Formatters;
+
+        inspectOpts?: {
+            hideDate?: boolean | number | null;
+            colors?: boolean | number | null;
+            depth?: boolean | number | null;
+            showHidden?: boolean | number | null;
```

Comment:
These types are accurate for the possible values in `debug`.

I'll note though, the types that `debug` allows are incorrect for `util.inspect()`